src/scss/partials/productListFooter/_ProductListFooter.scss

```diff
@@ -0,0 +1,4 @@
+.backToTop {
```

Comment:
proponuje inną strukturę do stylowania poszczególnego komponentu. Tutaj te style są globalne i mogą dotyczyć wszystkich komponentów czego nie chcemy

.ProductListFilter {
.backToTop {

}
.innaKlasaKtoraZnajdujeSieWProductListFooter {} itd.
}

poza .ProductListFilter {...} w tym pliku nie powinno znajdować się nic więcej. To dotyczy praktycznie wszystkich Twoich scss.